riscv/processor.cc

```diff
@@ -767,7 +767,9 @@ void processor_t::set_csr(int which, reg_t val)
 {
 #if defined(RISCV_ENABLE_COMMITLOG)
 #define LOG_CSR(rd) \
-  STATE.log_reg_write[((which) << 4) | 4] = {get_csr(rd), 0};
+  STATE.log_reg_write[((which) << 4) | 4] = { \
+      get_csr(rd, insn_t(0), false, true), \
+      0};
```

Comment:
I think you should undo this change, and...

Reply:
has been changed in push